test/unit/org/apache/cassandra/index/sai/cql/types/collections/maps/MapEntriesFrozenCollectionTest.java

```diff
@@ -32,6 +32,6 @@ public class MapEntriesFrozenCollectionTest extends IndexingTypeSupport
     public static Collection<Object[]> generateParameters()
     {
         DataSet<Map<Integer, Integer>> frozen = new CollectionDataSet.FrozenMapValuesDataSet<>(new DataSet.IntDataSet());
-        return generateParameters(new CollectionDataSet.MapEntriesDataSet<>(frozen));
+        return generateLiteralParameters(new CollectionDataSet.MapEntriesDataSet<>(frozen));
```

Comment:
Aside: So entries are literals even if the key and value are numeric?

Reply:
Entries are always literal but in this particular case, the index is literal because it is frozen.